docs/topics/settings.txt

```diff
@@ -271,3 +271,29 @@ It boils down to this: Use exactly one of either ``configure()`` or
 
     :doc:`The Settings Reference </ref/settings>`
```

Comment:
I'd leave the "See also" box at the bottom of the page, so just move your section about it. While you're there it looks like the "login_required" link is unused.